en/guide/mavlink_v1_compatibility_features.md

```diff
@@ -0,0 +1,97 @@
+# MAVLink v1.1 Compatibility and Features
+
+The MAVLink v1.1 release is the first release that requires a minimum set of messages to be supported to qualify as *MAVLink enabled*. 
+It will replaces a number of historically suboptimal messages with newer messages, while retaining full compatibility with v1.0.
+
+## Capability and Autopilot Version Discovery Mechanism
```

Comment:
If we're not maintaining this as "capability" info for v1.1 this section could become Autopilot discovery protocol (?)